client/components/token-field/token.jsx

```diff
@@ -10,7 +10,7 @@ var Token = React.createClass( {
 		value: React.PropTypes.string.isRequired,
 		displayTransform: React.PropTypes.func.isRequired,
 		onClickRemove: React.PropTypes.func,
-		status: React.PropTypes.oneOf( [ 'is-error', 'is-success' ] ),
+		status: React.PropTypes.oneOf( [ 'is-error', 'is-success', 'is-validating' ] ),
```

Comment:
I think this properties should be `error`, `success`, etc. It's something we've been wanting to fix for notices for some time.